advancedfx/export_agr2fbx.py

```diff
@@ -100,6 +100,8 @@ def execute(self, context):
 			if any(CameraData.name.startswith(c) for c in ("afxCam", "camera")):
 				# select camera
 				CameraData.select_set(1)
+				# scale camera

```

Comment:
What's the reason for only scaling the camera and not the rest?

Reply:
Devo basically sad this is an UE4 exclusive fix and other software doesn't care about it.
So I am just going to accept it, even though not happy about it.

Reply:
The exporter is mainly for UE4 usage, since other softwares profit more from normal FBX export.